community/flatpak-builder/APKBUILD

```diff
@@ -24,15 +26,9 @@ build() {
 		--libexecdir=/usr/lib/$pkgname \
 		--disable-static \
 		--with-dwarf-header=/usr/include/libdwarf
-
 	make
 }
 
-check() {
-	#make -k check
```

Comment:
Why remove it? Should be fixed someday instead of "ignore it"

Reply:
@fcolista (if i remember correctly) said it causes confusion to have check() defined with check disabled in options.

Reply:
Yup, @TBK also mentioned that